### PR TITLE
DON'T MERGE: Part of [SAIC-241]: create temp dir for check bandwidth step

### PR DESCRIPTION
**Pull request by realsystem**

### CODE REVIEW

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

File: cloudferrylib/os/actions/check_bandwidth.py [hunk 1]
```diff
@@ -29,6 +32,12 @@
 
 class CheckBandwidth(action.Action):
     def run(self, **kwargs):
+        runner = remote_runner.RemoteRunner(self.cloud.cloud_config.cloud.host,
+                                            self.cloud.cloud_config.cloud.ssh_user)
+
+        temp_dir_name = tempfile.mkdtemp(prefix=self.__class__.__name__)
```
Comment (roman-verchikov) [inline cloudferrylib/os/actions/check_bandwidth.py]: According to [documentation](https://docs.python.org/2/library/tempfile.html#tempfile.mkdtemp):
> tempfile.mkdtemp()
> Creates a temporary directory <...>
> <...>
> The user of mkdtemp() is responsible for deleting the temporary directory and its contents when done with it.
This doesn't seem right, we should not create anything locally. It should generate remote name for the target host without touching anything locally

Comment (realsystem) [inline cloudferrylib/os/actions/check_bandwidth.py]: sure

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Pull request closed without merging**